changelog.d/6799.misc

```diff
@@ -0,0 +1 @@
+[Create Room] Reduce some boilerplate with room state event contents
```

Comment:
Not sure about the changelog description, maybe I should have split the PR?

Reply:
I don't see anything wrong. It makes perfect sense to me